types/xml-flow/index.d.ts

```diff
@@ -0,0 +1,38 @@
+// Type definitions for xml-flow 1.0
+// Project: https://github.com/matthewmatician/xml-flow
+// Definitions by: Chris Lount <https://github.com/Warerebel>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// Minimum TypeScript Version: 3.8
+
+/// <reference types="node" />
+
+import { Readable } from "stream";
+import { EventEmitter } from "events";
+
+declare function flow(infile: Readable, options?: flow.parserOptions): EventEmitter;
+
+declare namespace flow {
+    enum frequency {
```

Comment:
There doesnt exist frequency on flow. It instead has consts for Always, never etc 
https://unpkg.com/browse/xml-flow@1.0.4/lib/xml-flow.js

Reply:
I agree, but do I have to change it?
The intention of these constants is to act as an enumeration in the JS. 
I model them as an enumeration in TS and this makes it explicitly clear to a user that they should be providing one of these enumerated values for the appropriate options?
Otherwise the type of these options is just number, or maybe -1 | 0 | 1, and both are less clear?